fastlane/Fastfile

```diff
@@ -13,6 +13,20 @@ ENV["DOWNLOAD_METADATA"]="./fastlane/download_metadata.swift"
 
 USER_ENV_FILE_PATH = File.join(Dir.home, '.simplenotemacos-env.default')
 
+workspace = 'Simplenote.xcworkspace'
+internal_scheme = 'Simplenote'
+app_store_scheme = 'Simplenote-AppStore'
+build_folder = 'build'
```

Comment:
nit: since those are constants, you should scream them in all-uppercase. 👮‍♂️  #manual-rubocoping

```suggestion
WORKSPACE = 'Simplenote.xcworkspace'
INTERNAL_SCHEME = 'Simplenote'
APP_STORE_SCHEME = 'Simplenote-AppStore'
BUILD_FOLDER = 'build'
```

(+ then of course update the references in rest of code)

Reply:
I DIDN'T KNOW YOU HAD TO SCREAM WHEN DEFINING CONSTANTS...

Would they remain uppercase if they became global values? That's a change I'm considering doing later while extracting some logic in a function.

```ruby
@WORKSPACE = ...

# or 

@workspace = ...
```

Reply:
Global variables should not be all uppercase. The convention-over-configuration rule in ruby about this is that all-uppercase means the value should be considered constant like if you `.freeze` it (in fact it's also good practice to `.freeze` constants, even though I admit I always forget).

The only reasonable reason to move those to be global variables instead is if you really need to make those values be variable, ie you plan to change their value during the script. And if you do, that means they won't be constant anymore so shouldn't be all-uppercase anymore.

That being said, global variables are frown upon in ruby like they are in many other languages — even if not as bad as class variables (I even think that rubocop considers `@@classvar` usage to be a big warning if not an error), but still not ideal. And I can't really see a reason why you'd need those values in your script in particular (workspace name and scheme name etc) to be (global) variables rather than constants in that case anyway?!